traffic_ops/traffic_ops_golang/cdni/shared.go

```diff
@@ -106,6 +107,25 @@ func GetCapabilities(w http.ResponseWriter, r *http.Request) {
 	api.WriteRespRaw(w, r, fciCaps)
 }
 
+func getBearerToken(r *http.Request) string {
+	if r.Header.Get("Authorization") != "" && strings.Contains(r.Header.Get("Authorization"), "Bearer") {
+		givenToken := r.Header.Get("Authorization")
+		if strings.Contains(givenToken, "access_token") {
+			givenToken = strings.Split(givenToken, "=")[1]
```

Comment:
We should probably do a length check before getting the second item from the array here.

Reply:
removed this one since having an "=" in there is uncommon per a comment above